rtree.go

```diff
@@ -88,24 +88,18 @@ func (s *dimSorter) Less(i, j int) bool {
 	return s.objs[i].bb.p[s.dim] < s.objs[j].bb.p[s.dim]
 }
 
-// split splits objects into slices of maximum k elements.
-func split(k int, objs []entry) [][]entry {
-	n := len(objs) / k
-	if len(objs)%k != 0 {
-		n++
-	}
+// walkPartitions splits objs into slices of maximum k elements and
+// iterates over these partitions.
+func walkPartitions(k int, objs []entry, iter func(parts []entry)) {
+	n := (len(objs) + k - 1) / k
```

Comment:
I always forget, can you add `// ceil(len(objs)/ k)`?

Reply:
I added the comment.